enginetest/queries/script_queries.go

```diff
@@ -1354,6 +1354,182 @@ CREATE TABLE tab3 (
 			},
 		},
 	},
+	{
+		Name: "last_insert_uuid() behavior",
```

Comment:
You also want to exercise these with the `INSERT INTO ... SELECT` form of insert statement. I'm pretty sure this panics on those queries as written.